shared/types/response-v3.ts

```diff
@@ -0,0 +1,117 @@
+import { BasicField, FormFieldDto, MyInfoChildAttributes } from './field'
+
+export type FieldResponsesV3 = Record<FormFieldDto['_id'], FieldResponseV3>
```

Comment:
I'm missing something. Why do we need to create this instead of reusing `FieldResponse[]` directly?